src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+function calculateSum(populationValue) {
+  return populationValue.reduce((accumulator, current) => {
+    return accumulator + current;
+  }, 0);
+}
+
+const populationElements = document.querySelectorAll('.population');
+
+const totalPopulationElement = document.querySelector('.total');
+const averagePopulationElement = document.querySelector('.average');
+
+const populationValues = [...populationElements].map((x) => {
```

Comment:
```suggestion
const populationValues = populationElements.map((x) => {
```
map создает новый массив

Reply:
но populationElements це колекція і вона немає методу map

Reply:
Але populationElements це колекція і вона не має методу map, тому я добавиви оператор spread

Reply:
а окей, сори тогда)